### PR TITLE
Machine updates

### PR DESCRIPTION
**Pull request by NinaWie**
Change 5681 "Checkpoint Charlie, Berlin Black Box Exhibit":
	Status from: available to: out-of-order

### CODE REVIEW

**Comment by NinaWie** [comment]
New machine 7184 named Thermalquellen von Saturnia submitted.

**Comment by jannisborn** [comment]
2024-05-19 22:30:02 INFO ======Location differ joblog from 2024-05-19 22:30:02=======
2024-05-19 22:58:22 INFO Snyder's Candy is available again
2024-05-19 22:58:24 INFO Location Delaware (21/125): Changes in 1/19 machines found.
2024-05-19 23:15:59 ERROR Geolocation failed for: Unknown Location 	 sub: , Arco
2024-05-19 23:40:02 INFO 332/440: Found machine to be added: Silverstone Museum
2024-05-19 23:41:10 INFO Location England (69/125): Changes in 1/440 machines found.
2024-05-20 00:07:20 INFO 
 Result: 2 changes, 2 new machines found and 0 machines retired
2024-05-20 00:22:17 ERROR Found 4 problems that require manual intervention
2024-05-20 00:22:17 INFO ======Location differ completed at 2024-05-20 00:22:17=======
2024-05-20 00:22:18 INFO Detected change in server_locations.json - push to github

**Comment by jannisborn** [comment]
2024-05-20 22:30:01 INFO ======Location differ joblog from 2024-05-20 22:30:01=======
2024-05-20 22:33:33 INFO Mount Roberts Tramway is available again
2024-05-20 22:33:59 INFO Location Alaska (3/125): Changes in 1/68 machines found.
2024-05-20 23:15:24 ERROR Geolocation failed for: Unknown Location 	 sub: , Arco
2024-05-21 00:05:36 INFO 
 Result: 1 changes, 1 new machines found and 0 machines retired
2024-05-21 00:20:19 ERROR Found 4 problems that require manual intervention
2024-05-21 00:20:19 INFO ======Location differ completed at 2024-05-21 00:20:19=======
2024-05-21 00:20:20 INFO Detected change in server_locations.json - push to github

**Comment by jannisborn** [comment]
No website updates today!

**Comment by NinaWie** [comment]
New machine 7186 named Rimini „La Ruota Panoramica“ submitted.

**Comment by jannisborn** [comment]
2024-05-22 22:30:02 INFO ======Location differ joblog from 2024-05-22 22:30:02=======
2024-05-22 23:15:33 ERROR Geolocation failed for: Unknown Location 	 sub: , Arco
2024-05-22 23:55:49 INFO 62/144: Found machine to be added: Grote Kerk 
2024-05-22 23:56:53 INFO Location Netherlands (94/125): Changes in 1/144 machines found.
2024-05-23 00:00:47 INFO 17/18: Found machine to be added: Please rename this country
2024-05-23 00:00:47 INFO Location Slovenia (109/125): Changes in 1/18 machines found.
2024-05-23 00:05:33 INFO 
 Result: 2 changes, 2 new machines found and 0 machines retired
2024-05-23 00:20:04 ERROR Found 4 problems that require manual intervention
2024-05-23 00:20:04 INFO ======Location differ completed at 2024-05-23 00:20:04=======
2024-05-23 00:20:04 INFO Detected change in server_locations.json - push to github

File: data/server_locations.json [hunk 1]
```diff
@@ -52665,6 +52665,52 @@
...
+            "type": "Feature",
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    19.699024,
+                    48.669026
+                ]
+            },
+            "properties": {
+                "name": "Please rename this country",
```
Comment (NinaWie) [inline data/server_locations.json:52702]: @jannisborn this seems to be a wrong new machine from the website, are we moving it to the skip.json file with the ignored machines?

**Comment by jannisborn** [comment]
2024-05-23 22:30:02 INFO ======Location differ joblog from 2024-05-23 22:30:02=======
2024-05-23 22:49:43 INFO 22/190: Found machine to be added: Castle Farms    
2024-05-23 22:51:39 INFO Location Michigan (13/125): Changes in 1/190 machines found.
2024-05-23 23:15:22 ERROR Geolocation failed for: Unknown Location 	 sub: , Arco
2024-05-23 23:17:27 INFO Virginia Beach Fishing Pier - Bait & Tackle shop is available again
2024-05-23 23:17:43 INFO Location Virginia (35/125): Changes in 1/121 machines found.
2024-05-24 00:04:46 INFO 34/46: Found machine to be added: Shin Kong Mitsukoshi Nanxi Store One
2024-05-24 00:04:56 INFO Location Taiwan (118/125): Changes in 1/46 machines found.
2024-05-24 00:06:04 INFO 
 Result: 3 changes, 3 new machines found and 0 machines retired
2024-05-24 00:20:41 ERROR Our machine Castle Farms in Michigan from Server shown as available but http://209.221.138.252/Details.aspx?location=418411 responds Internal Server Error (500)
2024-05-24 00:20:46 ERROR Found 4 problems that require manual intervention
2024-05-24 00:20:46 INFO ======Location differ completed at 2024-05-24 00:20:46=======
2024-05-24 00:20:47 INFO Detected change in server_locations.json - push to github

**Comment by jannisborn** [comment]
2024-05-24 22:30:02 INFO ======Location differ joblog from 2024-05-24 22:30:02=======
2024-05-24 23:05:37 INFO Buc-ee's is available again
2024-05-24 23:06:12 INFO Location Florida (24/125): Changes in 1/385 machines found.
2024-05-24 23:20:29 INFO 14/129: Found machine to be added: Grand Coulee Dam Visitor's Arrival Center
2024-05-24 23:21:43 INFO Location Washington (38/125): Changes in 1/129 machines found.
2024-05-25 00:06:47 INFO 
 Result: 2 changes, 2 new machines found and 0 machines retired
2024-05-25 00:21:34 ERROR Found 3 problems that require manual intervention
2024-05-25 00:21:34 INFO ======Location differ completed at 2024-05-25 00:21:34=======
2024-05-25 00:21:35 INFO Detected change in server_locations.json - push to github

**Comment by jannisborn** [comment]
New problems require attention.

**Comment by jannisborn** [comment]
2024-05-25 22:30:01 INFO ======Location differ joblog from 2024-05-25 22:30:01=======
2024-05-25 22:31:04 INFO Distance match - Seeems that machine Zoosiana at 5601 Hwy 90 E, Broussard already exists as: Zoo of Acadiana
2024-05-25 22:53:28 INFO Sea Quest Interactive Aquarium is available again
2024-05-25 22:54:05 INFO Location Colorado (15/125): Changes in 1/219 machines found.
2024-05-25 23:17:10 INFO Virginia Museum of History & Culture (Was Virginia Historical Society) is currently unavailable
2024-05-25 23:17:43 INFO Location Virginia (35/125): Changes in 1/121 machines found.
2024-05-25 23:19:46 INFO Tributes and Traditions Retro Arcade and Fascination is available again
2024-05-25 23:19:47 INFO Location New Jersey (37/125): Changes in 1/99 machines found.
2024-05-26 00:06:04 INFO 
 Result: 3 changes, 2 new machines found and 1 machines retired
2024-05-26 00:20:51 ERROR Found 3 problems that require manual intervention
2024-05-26 00:20:51 INFO ======Location differ completed at 2024-05-26 00:20:51=======
2024-05-26 00:20:51 INFO Detected change in server_locations.json - push to github

**Comment by jannisborn** [comment]
2024-05-26 22:30:02 INFO ======Location differ joblog from 2024-05-26 22:30:02=======
2024-05-26 23:38:20 INFO 177/441: Found machine to be added: Gruffalo & Friends Clubhouse
2024-05-26 23:40:51 INFO Location England (69/125): Changes in 1/441 machines found.
2024-05-27 00:06:41 INFO 
 Result: 1 changes, 1 new machines found and 0 machines retired
2024-05-27 00:21:37 ERROR Found 3 problems that require manual intervention
2024-05-27 00:21:37 INFO ======Location differ completed at 2024-05-27 00:21:37=======
2024-05-27 00:21:38 INFO Detected change in server_locations.json - push to github

**Comment by NinaWie** [comment]
New machine 7193 named Lambert’s Cafe 2 submitted.

**Comment by jannisborn** [comment]
2024-05-27 22:30:02 INFO ======Location differ joblog from 2024-05-27 22:30:02=======
2024-05-27 22:53:21 INFO Sea Quest Interactive Aquarium is currently unavailable
2024-05-27 22:53:57 INFO Location Colorado (15/125): Changes in 1/219 machines found.
2024-05-27 23:15:22 INFO Silver Lake Mall - Memory Lane Gems is currently unavailable
2024-05-27 23:15:41 INFO Location Idaho (33/125): Changes in 1/40 machines found.
2024-05-27 23:16:38 INFO Mabry Mill Restaurant and Gift Shop is currently unavailable
2024-05-27 23:17:25 INFO Location Virginia (35/125): Changes in 1/121 machines found.
2024-05-27 23:43:15 INFO Seeems that machine Emmerich Promenade Tourist Information at : Rheinpromenade 11, 46446 Emmerich am Rhein, Emmerich am Rhein already exists as: Rheinpromenade 11, 46446 Emmerich am Rhein, Germany
2024-05-28 00:05:19 INFO 
 Result: 3 changes, 0 new machines found and 3 machines retired
2024-05-28 00:19:51 ERROR Found 3 problems that require manual intervention
2024-05-28 00:19:51 INFO ======Location differ completed at 2024-05-28 00:19:51=======
2024-05-28 00:19:52 INFO Detected change in server_locations.json - push to github

**Review by jannisborn — APPROVED** [review]

**Pull request merged**